packs/tfe_fdo_nomad/templates/tfe.nomad.tpl

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+job [[ .tfe_fdo_nomad.job_name | quote ]] {
+  datacenters = [[ .tfe_fdo_nomad.datacenters  | toStringList ]]
+  namespace   = [[ .tfe_fdo_nomad.tfe_namespace | quote ]]
+  type        = "service"
+
+  group "tfe-group" {
+    count = [[ .tfe_fdo_nomad.tfe_group_count ]]
+    
+    spread {
+      attribute = "${node.unique.id}"
+    }
+
```

Comment:
```suggestion
```

Setting the [`spread`](https://developer.hashicorp.com/nomad/docs/job-specification/spread) block isn't necessary if you're just spreading across nodes, and it makes scheduling _much_ more expensive on large clusters.

> By default, when spread is omitted, the scheduler will attempt to place allocations from the same job on different nodes (and binpacked between jobs).